indico/modules/categories/controllers/management.py

```diff
@@ -58,8 +60,22 @@ def _category_query_options(self):
         return children_strategy,
 
     def _process(self):
+        show_all_events = request.args.get('show_all') == '1'
+        page_number = int(request.args.get('page', 1))
+        order_columns = {'start_dt': Event.start_dt, 'title': db.func.lower(Event.title)}
+        direction = db.desc if request.args.get('desc', '1') == '1' else db.asc
+        order_column = order_columns[request.args.get('order', 'start_dt')]
+        events_paginated = (Event.query.with_parent(self.category)
+                            .order_by(direction(order_column)))
```

Comment:
Actually sorting just by  the selected column is not enough, to keep the list stable you need to sort by `Event.id` too as a last criterion. As far as user-friendliness is concerned you could also use all unused sort options between the selected one and the ID but maybe that's overkill (it'd only affect the sorting if two events have the same title or start time)